pkg/skaffold/render/transform/transform.go

```diff
@@ -69,37 +90,12 @@ func validateTransformers(config []latestV2.Transformer) ([]kptfile.Function, er
 	for _, c := range config {
 		newFunc, ok := transformerAllowlist[c.Name]
 		if !ok {
```

Comment:
missing error return here

Reply:
Also, in `TestNewValidator_Error` which is failing due to this, use `t.CheckErrorContains` method instead of `t.CheckContains` to check for substring in error message. That handles nil pointer dereference error.

Reply:
Thanks for the suggestion. Done